### DISCUSSION TITLE
Increase Next.js SSR function memory

### DISCUSSION THREAD

**Discussion by msjonker** [vercel/vercel#9865] [Help] [answered]
### Problem Description

I've looked through the documentation, however I can't find anywhere to define the memory of the SSR function. https://github.com/vercel/vercel/discussions/4537 also asks about this, but it looks like the conversation was taken offline.

**Comment by msjonker** [comment, marked as answer]
Nevermind. My settings for the paths weren't being applied due to https://github.com/vercel/vercel/issues/9873

**Comment by dickyindra** [comment]
It's already in the vercel docs:
https://vercel.com/docs/concepts/projects/project-configuration#functions

Example `vercel.json`:
```
{
    "functions": {
        "src/pages/**/*": {
            "memory": 512
        }
    }
}
```
